src/NodaTime.Web/Markdown/2.0.x/versions.md

```diff
@@ -0,0 +1,4 @@
+@Title="Version history"
```

Comment:
I was wondering about that. I thought it might be worth keeping for consistency, but equally I'd be happy to be convinced otherwise. Now is the time to do it, of course.